### ISSUE TITLE
[Bug]: returning address of memory associated with a stack variable.

### ISSUE THREAD

**Issue by etiotto** [onnx/onnx-mlir#1209]
I saw this error when building with `clang`:

```
/Users/etiotto@ca.ibm.com/Source/onnx/onnx-mlir/src/Compiler/OnnxMlirCompiler.cpp:45:10: warning: address of stack memory associated with local variable 'val' returned [-Wreturn-stack-address]
  return val.c_str();
```

**Comment by AlexandreEichenberger** [comment]
@etiotto Is this being addressed?

**Comment by AlexandreEichenberger** [comment]
I see it is being addressed by #1210
